<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreguntaForm</class>
 <widget class="QWidget" name="PreguntaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tema:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblTema">
     <property name="styleSheet">
      <string notr="true">font: 900 italic 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Desconocido</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Concepto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="txtConceptos">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Termino:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbTerminos">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pregunta:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblContador">
     <property name="styleSheet">
      <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 127);
color: rgb(0, 0, 0);
font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
